### PR TITLE
Fullscreen keyboard control

### PR DESCRIPTION
**Pull request by Pharaphalyn**
Added the ability to go back to the previous image with left arrow key and to the next with right arrow in fullscreen mode.

### CODE REVIEW

File: src/components/pages/Stories.tsx [hunk 1]
```diff
@@ -66,6 +66,10 @@ export default function Stories() {
 
   const setIndex = React.useCallback((index: number) => dispatch(setFullscreenPost(index)), [dispatch])
 
+  const scrollToPrevious = React.useCallback(() => {
+    document.getElementById(`story-${prevIdx}`)?.scrollIntoView({ behavior: 'smooth' })
+  }, [nextIdx])
```
Comment (kurozenzen) [inline src/components/pages/Stories.tsx:71]: Probably a typo here. Should be `prevIdx` right?

File: src/components/player/media/Gif.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ export default function Gif(props: MediaProps) {
     height,
     detailsVisible,
     isFullscreen,
+    onBack,
```
Comment (kurozenzen) [inline src/components/player/media/Gif.tsx:21]: No need to unpack it here if it won't be used. This actually generates a warning.

File: src/components/player/media/Video.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ export default function Video(props: MediaProps) {
     isFullscreen,
     isActive,
     onFinished,
+    onBack
```
Comment (kurozenzen) [inline src/components/player/media/Video.tsx:22]: Same here. This generates a warning.

**Review by kurozenzen — CHANGES_REQUESTED** [review]
Really nice idea. Your code is fine as well. Just a few minor things to clean up.

Sorry for making the player so convoluted 😅. I really want to refactor it now. There shouldn't be that much duplicated code here...

File: src/components/pages/Stories.tsx (hunk 1 above)
Comment (Pharaphalyn) [reply src/components/pages/Stories.tsx]: You got me. I just mindlessly copied the scrollToNext function.

**Comment by kurozenzen** [comment]
Very nice. Thanks for contributing.

**Pull request merged**